service/templates/admin.html

```diff
@@ -458,6 +458,32 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									</table>
 
 								</md-card-content>
+								<md-card-title>
+									<md-card-title-text>
+										<span>Disabled GQL Queries</span>
+									</md-card-title-text>
+								</md-card-title>
+								<md-card-content>
+									<div>
+										<md-input-container class="control" style="width: 200px;">
+											<label>Operation name to disable</label>
+											<input type="text" default="ABC" ng-model="queryMapping.newQuery" />
+										</md-input-container>
+										<md-button ng-click="addDisabledGQLQuery()">
+											<i class="fa fa-plus"></i>
```

Comment:
This plus button isn't lined up / consistent with the rest of the page (we don't take legacy consistency too seriously but it does make it easier to understand).
![image](https://user-images.githubusercontent.com/26798134/165593172-d661b0cd-c616-462b-9c74-071f66c634b9.png)

I'd prefer if we went with the lined up green plus sign or the list format (example is CORS origin).

![image](https://user-images.githubusercontent.com/26798134/165593350-2cb0b3a3-fc7d-4b3d-a8b9-4e6a63069e73.png)

![image](https://user-images.githubusercontent.com/26798134/165593308-2e95d992-401f-4b25-8658-3ccb86a0fdd3.png)

Reply:
I changed the styles to align the buttons
<img width="423" alt="Screen Shot 2022-04-28 at 2 03 19 PM" src="https://user-images.githubusercontent.com/10734386/165818358-303f49d8-b165-455c-a314-5cf94351fbee.png">